hazelcast/src/main/java/com/hazelcast/security/IPermissionPolicy.java

```diff
@@ -53,9 +53,8 @@
      * {@code refreshPermissions} may be executed concurrently with {@code getPermissions}.
      *
      * @param permissionConfigs the new set of permissions to apply
-     * @return {@code true} if permissions were successfully refreshed, otherwise {@code false}.
      */
-    boolean refreshPermissions(Set<PermissionConfig> permissionConfigs);
+    void refreshPermissions(Set<PermissionConfig> permissionConfigs);
```

Comment:
Instead of returning a `boolean` to indicate success or failure, changed this method to `void`. Failures should be indicated by `RuntimeException`s.